### ISSUE TITLE
Minimize instead of quit

### ISSUE THREAD

**Issue by DemokritosTheAtomist** [inulute/perplexity-ai-app#25]
Dear devs,

Firstly many thank for this tool it is so great to be able to use Perplexity as an app !
It would be great if the application could minimize to the system tray when clicking the close button, instead of exiting. Alternatively, having this as an option in the settings menu would be ideal.

Peace !

**Comment by inulute** [comment]
> Dear devs,
> 
> Firstly many thank for this tool it is so great to be able to use Perplexity as an app ! It would be great if the application could minimize to the system tray when clicking the close button, instead of exiting. Alternatively, having this as an option in the settings menu would be ideal.
> 
> Peace !

Thanks for the suggestion. We'll try to add a toggle for it.

**Comment by ahh1539** [comment]
Yes! 100x this app would become much better instantly if I can use it in a similar fashion to chatgpt mac app. When I close the app I still want the option of having it run in the background without it showing up on the dock. Thanks all!

**Comment by tiagomaricate** [comment]
Exaclty!
Right now its on system tray and on dock.
Remove it from dock and allow only the system tray.
You can check the behavior from Chat GPT desktop app.

Agent window:
to open - use shortcut or system tray
close button - close
minimize button - close

App window:
to open - use shortcut or system tray
close button - close to the system tray
minimize button - minimizes to dock

Both use the same system tray icon.
